pointmatcher/DataPointsFilters/Geometry.h

```diff
@@ -0,0 +1,80 @@
+// kate: replace-tabs off; indent-width 4; indent-mode normal
+// vim: ts=4:sw=4:noexpandtab
```

Comment:
Same comment as the one in the source file: are these lines intended to be there?

Reply:
They are everywhere in the library :D